openwisp_users/apps.py

```diff
@@ -1,16 +1,24 @@
 from django.apps import AppConfig
 from django.conf import settings
 from django.utils.translation import ugettext_lazy as _
+from openwisp_users import settings as app_settings
 
 
 class OpenwispUsersConfig(AppConfig):
     name = 'openwisp_users'
     app_label = 'openwisp_users'
     verbose_name = _('Users and Organizations')
 
+    rest_framework_defaults = {
+        'DEFAULT_THROTTLE_RATES': {
+            'anon': app_settings.USERS_AUTH_THROTTLE_RATE,
```

Comment:
I'm not convinced of this.

Can't we just set this rate directly to our view and leave this setting on its own?

That way we can lower the default throttling rate for this endpoint, I think 20 requests a day should be more than enough. But we can't set such a low limit for all the endpoints.

Reply:
Sounds good, I also had the same feeling.